clusterctl/README.md

```diff
@@ -8,22 +8,30 @@ Read the [experience doc here](https://docs.google.com/document/d/1-sYb3EdkRga49
 
 ### Prerequisites
 
-Follow the steps listed at [CONTRIBUTING.md](https://github.com/kubernetes-sigs/cluster-api/blob/master/cluster-api/clusterctl/CONTRIBUTING.md) to:
-
-1. Build the `clusterctl` tool
-
-```
-go build
+1. Install [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) 
```

Comment:
Will this tool eventually spin up the minikube cluster for you?

Reply:
It will but perhaps that was pre-emptive of me to put it in the documentation before the code exists. Removed.